server/resp/src/main/java/org/infinispan/server/resp/RespServer.java

```diff
@@ -69,8 +71,8 @@ protected void startInternal() {
             builder.read(defaultCacheConfiguration);
             configuredValueType = builder.encoding().value().mediaType();
             if (cacheManager.getCacheManagerConfiguration().isClustered() &&
-                  !(builder.clustering().hash().keyPartitioner() instanceof CRC16HashFunctionPartitioner)) {
-               log.warn("Clustered RESP server should use CRC16HashFunctionPartitioner for key partitioning.");
+                  !(builder.clustering().hash().keyPartitioner() instanceof RESPHashFunctionPartitioner)) {
+               log.warn("Clustered RESP server should use RESPHashFunctionPartitioner for key partitioning.");
```

Comment:
Should we enforce this?

Reply:
Updated, so now it is required to utilize the function. However, this might not fit well when using a default configuration.

Reply:
Yes, we have to require it. The error should be pretty clear what needs to be done at startup, so I see no reason to having a template break this. The user would have to define their cache to use the proper hash function is all.